modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -401,19 +401,23 @@ void GDScriptWorkspace::_get_owners(EditorFileSystemDirectory *efsd, String p_pa
 	}
 }
 
-Node *GDScriptWorkspace::_get_owner_node(String p_path) {
-	Node *owner_node = NULL;
+Node *GDScriptWorkspace::_get_owner_scene_node(String p_path) {
+	Node *owner_scene_node = NULL;
 	List<String> owners;
 
 	_get_owners(EditorFileSystem::get_singleton()->get_filesystem(), p_path, owners);
 
-	if (owners.size() > 0) {
-		NodePath owner_path = owners[0];
-		Ref<PackedScene> owner_res = ResourceLoader::load(owner_path);
-		owner_node = owner_res->instance(PackedScene::GEN_EDIT_STATE_DISABLED);
+	for (int i = 0; i < owners.size(); i++) {
+		NodePath owner_path = owners[i];
+		RES owner_res = ResourceLoader::load(owner_path);
+		if (owner_res->get_class() == "PackedScene") {
```

Comment:
Using `Object::cast_to<PackedScene>(owner_res.ptr())` instead of the name of the class for type checking should be better for future refactor.